data/json/recipes/recipe_deconstruction.json

```diff
@@ -18,7 +18,14 @@
     "type": "uncraft",
     "time": "30 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "splinter", 1 ] ] ]
+    "components": [ [ [ "wooden_bead", 30 ] ] ]
```

Comment:
The bead necklace has the exact same volume as the bracelet. Either they should have the same number of beads or the necklace volume should be increased. (Also, i'm unsure how one would extract intact beads with a hammer. CUT maybe?)

Reply:
Here I just changed because it seemed strange to get the splinter from the necklace or bracelet, then I just thought it should be natural to get the beads back. I didn't even notice that it was using a hammer. I just changed the splinter part but I can change the hammer too haha

Reply:
And here I just got back what was used in the crafting, then should I change the crafting recipe too? Change the volume then?

Reply:
I would increase the volume (and corresponding weight) of the necklace. It doesn't make sense that it would be the exact same size as a bracelet.